launch_with_pydiscovery.cmd

```diff
@@ -0,0 +1,25 @@
+@if not defined _echo echo off
+set "lscp-path=%cd%"
+
+rem https://stackoverflow.com/questions/22352793/reading-a-registry-value-to-a-batch-variable-handling-spaces-in-value
+for /f "usebackq tokens=2,*" %%h in (
+  `"reg query "HKLM\SOFTWARE\Kitware\CMake\Packages\pico-sdk-tools" 2>NUL | find /i "SDK""`

```

Comment:
I would suggest that you check the uninstall entry to detect the path of pico-setup-windows, e.g. `HKLM\Software\Microsoft\Windows\CurrentVersion\Uninstall\Raspberry Pi Pico SDK v*`, value `InstallPath`.

I guess it'll be harder to enumerate keys to find the latest version this way, but maybe consider a PowerShell snippet here?

Reply:
Thanks for the suggestion. I have changed the script accordingly. I can share if it is of interest.
PowerShell would be nice. I will consider changing the complete script to PowerShell instead of using just a snippet.

Reply:
If you are interested in the PowerShell version of the launcher: It is available here:
https://github.com/BitRolher/pico-project-generator/tree/BitRolher-WinLauncherPSversion